src/app/api/events/status/route.ts

```diff
@@ -0,0 +1,33 @@
+import { NextResponse } from "next/server";
+import prisma from "@/../lib/prisma";
+
+export async function GET(request: Request) {
+  try {
+    console.log("Here");
+    const result = await prisma.event.findFirst({
+      orderBy: {
+        endTime: "desc",
+      },
+      select: {
+        id: true,
+        isActive: true,
+      },
```

Comment:
Can we add more fields?
```suggestion
      select: {
        id: true,
        isActive: true,
        name: true,
        startTime: true,
        endTime: true,
        timerDuration: true,
      },
```